cli/args/flags.rs

```diff
@@ -3764,6 +3764,7 @@ fn env_file_arg() -> Arg {
     .default_missing_value(".env")
     .require_equals(true)
     .num_args(0..=1)
+    .action(ArgAction::Append)
```

Comment:
I think the help text here could be improved to mention what happens when multiple files are specified - I assume the vars from first files are overridden from vars from the second file, but not sure

Reply:
@bartlomieju , so actually, the var are not overridden. The crate dotenvy's `from_filename` function does this work by preserving the already included variables. Below are the words from that docs for your reference:
```plaintext
Loads environment variables from the specified file.

If variables with the same names already exist in the environment, then their values will be preserved.

Where multiple declarations for the same environment variable exist in your .env file, the first one is applied.
```
So, for this, I applied the last file first and then those are preserved as we go backward.

Reply:
Okay good to know. Since the linked issue is about parity with Node.js, we should verify how Node behaves here and apply the same behavior. Can you please check that and report back?

Reply:
@bartlomieju  This is what the [documentation](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs) says regarding `--env-file` : 

```plaintext
you can pass multiple --env-file arguments. Subsequent files override pre-existing variables defined in previous files.

Note: if the same variable is defined in the environment and in the file, the value from the environment takes precedence.
```
### Example Code:
```bash
node --env-file=.env --env-file=.development.env app.js
``` 

I did refer to this document before implementation, so I suppose the behavior is satisfied.

Reply:
@bartlomieju , Here's the updated help text:
```bash
      --env-file[=<FILE>]             Load environment variables from local file
                                        Only the first environment variable with a given key is used.
                                        Existing process environment variables are not overwritten, so if variables with the same names already exist in the environment, their values will be preserved.
                                        Where multiple declarations for the same environment variable exist in your .env file, the first one encountered is applied. This is determined by the order of the files you pass as arguments.
```